README.md

```diff
@@ -12,194 +12,129 @@ shared. These resources have a different lifecycle than resources in workspaces.
 
 ## Implementing a Landing Zone
 
-### Landing Zone Definition Factories and Landing Zone Definitions.
+Landing zone creation process is implemented as a [Stairway](https://github.com/DataBiosphere/stairway) flight represented by the following class `CreateLandingZoneFlight`. This flight consists of different steps one of which is step which 
+represents sub-flight to create Azure resources. This sub-flight is represented by the following class `CreateLandingZoneResourcesFlight`. This class utilizes specific steps' provider to define the list of steps required to create Azure resources.
 
-Landing zones are implemented using the factory pattern; the factory creates *Landing Zone Definitions* (LZDs).
+Below are lists of changes required to introduce new landing zone:
+1) Implement specific steps which are responsible for Azure resource creation (see example below for `CreateVnetStep`)
+2) Implement specific steps' definition provider (by implementing `StepsDefinitionProvider` interface)
```

Comment:
Suggested: Create an implementation of `StepsDefinitionProvider` to define the list of Azure resource creation steps.